python/src/GeneralizedExtremeValueFactory_doc.i.in

```diff
@@ -49,3 +51,362 @@ Returns
 -------
 distribution : :class:`~openturns.GeneralizedExtremeValueFactory`
     The estimated distribution"
+
+// ----------------------------------------------------------------------------
+
+%feature("docstring") OT::GeneralizedExtremeValueFactory::buildMethodOfLikelihoodMaximization
+"Estimate the distribution with maximum likelihood method.
+
+The parameters are estimated thanks to a numerical optimization of the likelihood:
+
+.. math::
+
+    \ell (\mu, \sigma, \xi) = -m \log{\sigma} - (-1 + 1 / \xi) \sum_{i=1}^m \log{\left[ 1 + \xi \left( \frac{z_i - \mu}{\sigma} \right) \right]} - \sum_{i=1}^m \left[ 1 + \xi \left( \frac{z_i - \mu}{\sigma} \right) \right]^{-1 / \xi}
+
+if :math:`1+\xi \left( \frac{z_i - \mu}{\sigma} \right) > 0 \forall i`, and by:
+
+.. math::
+
+    \ell (\mu, \sigma) = -m \log{\sigma} - \sum_{i=1}^m \left( \frac{z_i - \mu}{\sigma} \right) - \sum_{i=1}^m \exp \left\{ \frac{z_i - \mu}{\sigma} \right\}
+
+if :math:`\xi = 0`
```

Comment:
This does not make sense to me, because the if conditions are not alternatives: both can be true, and both can be false! Can you explain @adutfoy ?

Reply:
Yes. The first condition is :
:math:`\xi \neq 0`. The formula is valid for any :math:`z_i` such that :math:`1+\xi \left( \frac{z_i - \mu}{\sigma} \right) > 0

Reply:
Il faut écrire: 
If :math:\xi \neq 0, then 
.. math::

    \ell (\mu, \sigma, \xi) = -m \log{\sigma} - (-1 + 1 / \xi) \sum_{i=1}^m \log{\left[ 1 + \xi \left( \frac{z_i - \mu}{\sigma} \right) \right]} - \sum_{i=1}^m \left[ 1 + \xi \left( \frac{z_i - \mu}{\sigma} \right) \right]^{-1 / \xi}

provided that :math:`1+\xi \left( \frac{z_i - \mu}{\sigma} \right) > 0 \forall i`

Reply:
ok